doc/guides/security_release_process.md

```diff
@@ -1,61 +1,83 @@
+  unlikely to attract much additional comment. ***LINK TO BLOG PR AND POST***
 
 * [ ] Open an issue in the build working repository with a notification of the
   date for the security release.  Use this issue to co-ordinate with the build
   team to ensure there will be coverage/availability of build team resources the
   day of the release. Those who volunteer from the build WG should be available
-  in node-build during the release in case they are needed by the individual
-  doing the release.
-
-* [ ] Send an email to the docker official image
-  [maintainers](https://github.com/docker-library/official-images/blob/master/MAINTAINERS)
-  with an FYI that security releases will be going out on the agreed date.
-
-* [ ] Open an issue in the [docker-node](https://github.com/nodejs/docker-node)
-  repo and get one or more volunteers to be available to review the PR to update
-  Node.js versions in the docker-node repo immediately after the release.
-
-* [ ] Call on the sec release volunteer(s) to start integrating the PRs, running
-  the CI jobs, and generally prepping the release.
+  in node/build during the release in case they are needed by the individual
+  doing the release. ***LINK TO BUILD ISSUE***
 
 ## Release day
 
-* [ ] Co-ordinate with the Release team members and keep up to date on progress.
-  Get an guesstimate of when releases may be ready and send an FYI to the docker
-  official image
-  [maintainers](https://github.com/docker-library/official-images/blob/master/MAINTAINERS).
-
-* [ ] When the releases are promoted, ensure the final announce goes out as per
-  the
-  [security_announcement_process](https://github.com/nodejs/security-wg/blob/master/processes/security_annoucement_process.md).
+* [ ] The releaser(s) run the release process to completion.
+
+* [ ] Send post-release announcement as a reply to the
+  original message in https://groups.google.com/forum/#!forum/nodejs-sec
+  ***LINK TO EMAIL***
+  
+* [ ] Update the blog post in
+  https://github.com/nodejs/nodejs.org/tree/master/locale/en/blog/vulnerability
+  with the information that releases are available and the full
+  vulnerability details. Keep the original blog content at the
+  bottom of the blog. Use this as an example:
+  https://github.com/nodejs/nodejs.org/blob/master/locale/en/blog/vulnerability/june-2016-security-releases.md.
+  Make sure to update the date in the slug so that it will move to
+  the top of the blog list, and not that it updates the
+  banner on Node.js org to indicate the security release(s) are
+  available. ***LINK TO PR***
+
+  *Note*: If the release blog obviously points to the people having caused the
+  issue (for example when explicitly mentioning reverting a commit), adding
+  those people as a CC on the PR for the blog post to give them a heads up
+  might be appropriate.
+
+* [ ] Email foundation contact to tweet out nodejs-sec announcement from
+  foundation twitter account. FIXME - who is this contact?
```

Comment:
@nodejs/tsc who is this contact? Is this even correct anymore? I think the twitter account may be in process of becoming a direct TSC responsibility again?